blog/2022-05-11-Chia_Releases_NFT0.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: chia-releases-nft0
+title: Chia Releases NFT0
+description: Overview of the NFT0 alpha NFT standard, how to mint NFTs, and how to view them on MintGarden.io
+authors: clydewallace22
+tags: [overview, nft, nft0, announcement]
+image: /img/blog/nft0_cc.png
+---
+![Chia Releases NFT0](/img/blog/nft0_cc.png)
```

Comment:
Let's swap the two images and use our screenshot as the header. Since it shows up on Twitter etc.

Reply:
Swapped images and headings